readme.txt

```diff
@@ -242,6 +242,33 @@ Yes, through the use of our commercial addon called [Recurring Payments](https:/
 13. Pay with PayPal modal
 
 == Changelog ==
+
+= 2.11.4, December 27, 2021 =
+* New: Add EDD notifications to EDD screens.
+* New: Add product education to EDD settings screens.
```

Comment:
This should read something more focused on the user experience, maybe:
`* New: Re-organized some settings locations to make them more predictable for users.`